configargparse.py

```diff
@@ -223,12 +223,23 @@ def parse(self, stream):
                 "yaml.load('%s') returned type '%s' instead of 'dict'." % (
                 getattr(stream, 'name', 'stream'),  type(parsed_obj).__name__))
 
+        def _kvgen(d, pfix=None):
```

Comment:
please add method docs like you have in _set_lvlkey to explain what feature this is supporting, and switch to longer but clearer names (eg. 'pfix' => 'prefix',  'npfix' => 'full_key' or similar)